app/views/pages/get-started/working-with-the-code-and-files.liquid

```diff
@@ -0,0 +1,178 @@
+
+Let’s assume the first created instance will be for development, thus we can name it `dev`. So, the example pos-cli command would look like this:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli env add dev --email myemail@example.com —-url https://my-instance.staging.oregon.platform-os.com
+</code></pre>
+
+After running this command, you’ll get asked for your Partner Portal password to authenticate. Providing the correct password will be confirmed by pos-cli:
+
+```bash
+Environment <your instance url> as <environment> has been added successfully.
+```
+
+
+## Upload your code to an instance
+
+When we have an instance, and pos-cli is configured to connect with it, we would like to send our code to be available under the instance URL.
+
+This process – called deployment – is a straight-forward operation and you just need to use the following command in the root of your project folder:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy &lt;environment&gt;
+</code></pre>
+
+So, if we named the alias to our instance as <code>dev</code>, the command would look like:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy dev
+</code></pre>
+
+The progress indicator is shown with the deployment status and once it finishes, a message is displayed:
+
+```bash
+Deploy succeeded after X seconds
+```
+
+After this you shall be able to enter the URL of your instance in the browser and it will be showing the example page that we’ve just deployed.
+
+
+### Testing the deployment process
+
+To test out the whole deployment process, let’s make some changes in the code.
+
+If you’re following this guide, you should be able to find a file named index.html.liquid in your codebase. Open it up in your code editor of choice and change any line. For example let’s locate a line with the welcome message:
+
+```html
+Welcome to your Demo instance!
+```
+
+Try changing it into something more personal like:
+
+```html
+This is my first platfromOS instance!
+```
+
+Save the file and – as we learned – send the codebase to the instance by running the deploy command in the command line again:
+
+<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+pos-cli deploy dev
+</code></pre>
+
+After visiting your instance URL or refreshing previously opened page, you should see the new message there.
+
+Ok, this works, but having to manually deploy each change would take far too much time. You wouldn’t want to enter any commands or wait so long for the pos-cli to send all of the files in your codebase each time you change a single line of your code. Because this is what the deploy command is doing – sending all the files in the codebase to the instance.
```

Comment:
Oh, I haven't know this. Are we sure this works? Cause my Template deployments always takes forever. :D

I'll update this then not to mentioning the time, it would make it simpler, thanks!